.changeset/good-pugs-check.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": minor
+---
+
+Add Overlays and DocumentOverlayCache implementation
```

Comment:
This is not a user-facing change so I would drop the changeset. It might even be misleading as we do not use the overlays yet.